sdk/src/server-api/sc/framework/plugins/ActionTimeout.java

```diff
@@ -67,7 +67,7 @@ public synchronized boolean didTimeout() {
     }
 
     if (this.canTimeout()) {
-      logger.info("Time needed: " + String.valueOf(this.stopTimestamp - this.startTimestamp));
+      logger.debug("Time needed: " + String.valueOf(this.stopTimestamp - this.startTimestamp));
```

Comment:
Don't you think this might be valuable information to check how long a client takes?

Reply:
yes, but the Timeout object shouldn't log this at all tbh.

Reply:
okay, fair.

Reply:
Fixed with 4c0182f825ffbc2c90ab6f00cc23b4fa3c931dba